pysgrid/sgrid.py

```diff
@@ -351,6 +815,20 @@ def get_cell_node_lat_lon(self):
             node_lon = self.nc[node_lon_var]
         return node_lon, node_lat
 
+    def get_cell_edge1_lat_lon(self):
```

Comment:
Now sure we need the `lat_lon` `edge1` is by definition the positions. (And they might be metric x, y as far as I known.)